packages/node_modules/@ciscospark/react-redux-spark/src/component.test.js

```diff
@@ -7,14 +7,44 @@ import {createMockStore} from '@ciscospark/react-redux-spark-fixtures';
 import SparkComponent from './component';
 
 describe('spark component', () => {
-  it('renders correctly', () => {
-    const store = createMockStore();
-    const component = renderer.create(
+  let sparkInstance, store, component;
+
+  beforeEach(() => {
+    store = createMockStore();
+    sparkInstance = {
+      authorization: 'authorization',
+      logger: 'logger',
+      people: 'people',
+      phone: 'phone',
+      rooms: 'rooms',
+      internal: {
+        conversation: 'conversation',
+        feature: 'feature',
+        flag: 'flag',
+        mercury: 'mercury',
+        presence: 'presence',
+        search: 'search',
+        team: 'team'
+      }
+    };
+    component = renderer.create(
       <Provider store={store}>
```

Comment:
I definitely prefer to not use the `Provider` here. any ideas to make this work with mock `store`?

Reply:
that way to mock the spark instance, which is huge. I would like to add to our new testing widget design epic.